@here/harp-examples/src/object-picking.ts

```diff
@@ -173,8 +186,30 @@ export namespace PickingExample {
 
         // snippet:datasource_object_picking_1.ts
         canvas.addEventListener("mousedown", event => {
-            handlePick(mapView, event.pageX, event.pageY);
+            lastEventCoords = { x: event.pageX, y: event.pageY };
+        });
+        canvas.addEventListener("touchstart", event => {
+            if (event.touches.length !== 1) {
+                return;
+            }
+            lastEventCoords = { x: event.touches[0].pageX, y: event.touches[0].pageY };
```

Comment:
MINOR: Using `event.pageX` and `event.pageY` can cause issues if the canvas isn't covering the whole screen. It's better to use `event.clientX - boundingRect.left` and `event.clientY - boundingRect.top`. See `MapControls.getPointerPosition()`.

Reply:
done.